pkg/odo/cli/url/list.go

```diff
@@ -18,10 +19,12 @@ import (
 	"github.com/openshift/odo/pkg/lclient"
 	"github.com/openshift/odo/pkg/log"
 	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/openshift/odo/pkg/odo/cli/component"
 	"github.com/openshift/odo/pkg/odo/genericclioptions"
-	"github.com/openshift/odo/pkg/odo/util"
+	odoUtil "github.com/openshift/odo/pkg/odo/util"
```

Comment:
better to use `odoutil` similar to how we do it in different packages. Instead of `odoUtil`

Reply:
I feel `odoutil` looks weird as it is not a camel case word. I think we should change all occurrences to `odoUtil`.

Reply:
Please refer to https://blog.golang.org/package-names

Package names should be short and sweet and not use camel case.

Example:

Package names
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns, such as:

time (provides functionality for measuring and displaying time)
list (implements a doubly linked list)
http (provides HTTP client and server implementations)
The style of names typical of another language might not be idiomatic in a Go program. Here are two examples of names that might be good style in other languages but do not fit well in Go:

computeServiceClient
priority_queue
A Go package may export several types and functions. For example, a compute package could export a Client type with methods for using the service as well as functions for partitioning a compute task across several clients.

Reply:
for example, below we import ktemplates, we do not use camelCase for that as kTemplates